packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,134 @@ section: components
+<div class="ws-content-table">
 
-Toast notifications must always have a close button so the user can dismiss them. You may also include an optional time-out to dismiss the alert automatically after a period of time. Eight (8) seconds is a recommended timeout. Alerts should not dismiss while the user is hovering over them.
+| **Do** | **Don’t** |
+|--------|-----------|
+| Account created| Your account was successfully created.|
 
-#### When not to use
-Do not use a toast alert when an event requires immediate action or for synchronous events like form validation.
+</div>
 
-### Stacking multiple alerts
-Multiple alerts will stack. The most recent alert should appear at the top and others will push down. When alerts are deleted, any space above will be filled by pushing remaining alerts up in the stack order.
+<br /> 
 
-An [alert group](/components/alert-group) component allows a set of alerts to be properly positioned and stacked as shown below.
+- Be concise. In one-line toast alerts, use fragments instead of sentences if possible.
 
-![alert group](./img/toast-notification-multiple.png)
+<div class="ws-content-table">
 
-### Examples
+| **Do** | **Don’t** |
+|--------|-----------|
+| Message sent| Your message has been sent.|
 
-#### Toast alert placement
-Toast alerts always stack in the upper right of a page.
+</div> 
 
-<img src="./img/Toast-alerts-desktop.png" alt="toast alerts in a page" />
+<br />
 
+- In toast alerts containing links, describe how the user can navigate through the UI to perform the action. Learn more about how descriptive text supports accessibility in the next section.
 
-#### Inline alert placement
-If an alert message applies globally to the content on a page, place the alert in the page header area just below the title.
+| **Do** | **Don’t** |
+|--------|-----------|
+| From the **Settings** tab, click **View logs.**| **View logs**|
 
-<img src="./img/Inline-alert-page-level.png" alt="global inline alert in a page" />
+## Accessibility considerations
+Toast alerts present special challenges for accessibility. Screen readers announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This makes it difficult for these users to directly act on the alert itself.
 
+In any toast alert containing links for further user action, include text that describes how the user can navigate through the UI to perform the action.
 
-But if the alert applies only to specific content, you might consider placing the alert inline with page content to call attention to the error.
+For example, this toast alert is not accessible because it does not include text that explains to the user where to go within the UI to perform the action: 
 
-<img src="./img/Inline-alert-specific-context.png" alt="inline alert inline with content" />
+<img src="./img/alert-no-description.png" alt="alert without descriptive text" width="600px"/>
```

Comment:
confused because this image has a shadow, but the one right under it doesn't haha looks a little bit weird

Reply:
Ah, that was because one was positioned at the bottom of the art board, cutting off the shadow. I can recreate.